code/src/components/TaskInput.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable react/jsx-closing-bracket-location */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import tasks from 'reducers/tasks';
+import { InnerWrapper, OuterWrapper } from 'styles/GlobalStyles';
+
+export const TaskInput = () => {
+  // useState to set new tasks
+  const [inputValue, setInputValue] = useState('');
+  // useDispatch to add new task to list
+  const dispatch = useDispatch();
+
+  const onFormSubmit = (event) => {
+    // Prevent page from reloading
+    event.preventDefault();
+    // Creating a new task as an object
+    const newTask = {
+      id: Date.now().toString(),
+      text: inputValue,
+      complete: false,
+      created: Date.now().toString(),
+    };
+    dispatch(tasks.actions.addTask(newTask));
+    // Clear input
+    setInputValue('');
+  };
+
+  return (
+    <OuterWrapper>
+      <InnerWrapper>
+        <form onSubmit={onFormSubmit}>
+          <button type="submit">+</button>
+          <label htmlFor="a">
```

Comment:
How come you are using `htmlFor="a"`? I think you should connect it to the form the label is associated with using an id on the form